charts/microgateway/README.md.gotmpl

```diff
@@ -376,6 +377,21 @@ Finally, apply the Helm chart configuration file with `-f` parameter.
   helm upgrade -i microgateway airlock/microgateway -f custom-values.yaml -f env-variables.yaml
   ```
 
+## Extra Volumes
+The Helm Chart allows you to define extra volumes which can then be used in the Microgateway. 
+The configuration of such additional volumes could look like this, for example: 
```

Comment:
> The configuration of such additional volumes could look like this, for example: 
I would cross out ", for example". Cloud look like already says, that this is an example.